contracts/cairo/deploy.sh

```diff
@@ -6,24 +6,24 @@ PURPLE='\033[1;34m'
 PINK='\033[1;35m'
 COLOR_RESET='\033[0m'
 
-HERODOTUS_FACTS_REGISTRY=0x01b2111317EB693c3EE46633edd45A4876db14A3a53ACDBf4E5166976d8e869d
-MM_ETHEREUM_WALLET=0xE8504996d2e25735FA88B3A0a03B4ceD2d3086CC
-NATIVE_TOKEN_ETH_STARKNET=0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
+if [ -f ./contracts/cairo/.env ]; then
+    echo "Sourcing .env file..."
+    source ./contracts/cairo/.env
+else
+    echo "Error: .env file not found!"
+    exit 1
+fi
 
-cd "$(dirname "$0")"
+# Starkli implicitly utilizes these environment variables, so every time we use Starkli,
+# we avoid adding flags such as --account, --keystore, and --rpc.
+export STARKNET_ACCOUNT=$STARKNET_ACCOUNT
+export STARKNET_KEYSTORE=$STARKNET_KEYSTORE
+export STARKNET_RPC=$STARKNET_RPC
```

Comment:
This is not a good idea is better to have them in the env files

Reply:
Actually they are set as environment variables in the `.env` file. However, for some reason, if I dont make the `exports` starkli it doesn't recognize them (If I print them using echo, the variables are set in both cases). 

I can remove the `export` statements, but I would have to add each flag with the corresponding environment variable.

### Declare: 
from this
```
ESCROW_CLASS_HASH=$(starkli declare --watch target/dev/yab_Escrow.contract_class.json)
```
to this
```
ESCROW_CLASS_HASH=$(starkli declare \
    --account $STARKNET_ACCOUNT \
    --keystore $STARKNET_KEYSTORE \
    --rpc $STARKNET_RPC \
    --watch target/dev/yab_Escrow.contract_class.json)
```

### Deploy:
 from this
```
 ESCROW_CONTRACT_ADDRESS=$(starkli deploy --watch $ESCROW_CLASS_HASH \
    $SN_PROXY_OWNER \
    $HERODOTUS_FACTS_REGISTRY \
    $ETH_CONTRACT_ADDR \
    $MM_ETHEREUM_WALLET \
    $MM_SN_WALLET_ADDRESS \
    $NATIVE_TOKEN_ETH_STARKNET)
```
to this
```
ESCROW_CONTRACT_ADDRESS=$(starkli deploy \
    --account $STARKNET_ACCOUNT \
    --keystore $STARKNET_KEYSTORE \
    --rpc $STARKNET_RPC \
    --watch $ESCROW_CLASS_HASH \
    $SN_PROXY_OWNER \
    $HERODOTUS_FACTS_REGISTRY \
    $ETH_CONTRACT_ADDR \
    $MM_ETHEREUM_WALLET \
    $MM_SN_WALLET_ADDRESS \
    $NATIVE_TOKEN_ETH_STARKNET)
```